components/ws-proxy/pkg/proxy/proxy.go

```diff
@@ -69,7 +69,7 @@ func (p *WorkspaceProxy) Server() (*http.Server, error) {
 		return nil, err
 	}
 	theiaRouter, portRouter := p.WorkspaceRouter(r, p.WorkspaceInfoProvider)
-	installTheiaRoutes(theiaRouter, handlerConfig, DefaultRouteHandlers(p.WorkspaceInfoProvider))
```

Comment:
@chris I've removed `DefaultRouteHandlers`, got tired of navigating back and forth